app/views/main.scala.html

```diff
@@ -50,7 +50,8 @@
   optimizelyAudience: Option[String] = None,
   pageHeader: Option[Html] = None,
   script: Option[Html] = None,
-  hidePtaAccountNav: Boolean = false
+  hidePtaAccountNav: Boolean = false,
+  displayTimeoutDialog: Boolean = true
```

Comment:
Can't see where this is being set too false, do you need it?

Reply:
yes its not used as only unauthenticated main is used for session expired page where we dont want to display timeout dialog so i've removed it